models/users.js

```diff
@@ -412,6 +414,34 @@ const getUsersBasedOnFilter = async (query) => {
     const filteredUserDocs = userDocs.filter((doc) => !doc.roles?.archived);
     return filteredUserDocs;
   }
+
+  const roleQuery = query.role;
```

Comment:
We can use destructuring here. Shorter code 😃

Reply:
done